### DISCUSSION TITLE
Simple training and validation set?

### DISCUSSION THREAD

**Discussion by jmrichardson** [pycaret/pycaret#1979] [Q&A] [answered]
Hi, I have provided a train and validation set in setup where I just want to train on the training set with cross validation:

```
        exp_name = setup(
            data=train,
            test_data=valid,
            target='target',
            transformation=True,
            transformation_method='yeo-johnson',
            feature_selection=True,
            feature_selection_threshold=.8,
            feature_selection_method='boruta',
            data_split_shuffle=False,
            silent=True
        )
```

However, it is doing 10 fold cross validation?  I am not sure why folds are being created when I have provided my train and test set?  I just want to do a simple train on the training set with cross validation on the test_data.  Thanks for any help

**Comment by jmrichardson** [comment]
Ok, I found in the guide that test_data is held out data and not used for cross validation (was confused by verbiage in the API docs).   However, if I make train_size=1.0, I get the following error:

`ValueError: test_size=0.0 should be either positive and smaller than the number of samples 6650 or a float in the (0, 1) range`

I don't want any test data and it all to be used for training/CV.

**Comment by ngupta23** [comment, marked as answer]
> However, it is doing 10 fold cross validation? I am not sure why folds are being created when I have provided my train and test set? I just want to do a simple train on the training set with cross validation on the test_data. Thanks for any help

You can set `cross_validate=False` and then it will not create the folds on the `train` split

**Reply by jacobdang** [reply]
cross_validate=False  option is not available for tune_model. How can we do a simple train-validation split without cross-validation for the tune_model function? Is it supported?
